scripts/src/main/resources/scripts/command/ide

```diff
@@ -320,7 +320,7 @@ function doUpdateSoftware() {
   local tool_path
   if [ -d "${DEVON_IDE_HOME}/software" ]
   then
-    for tool_path in "${DEVON_IDE_HOME}/software"/*
+    for tool_path in "${DEVON_IDE_HOME}/software"/* "${DEVON_IDE_HOME}/software/extra"/*
```

Comment:
Mhm, I see your point here but this seems dangerous and might contradict with what we actually plan with `extra` folder (see e.g. #683).

IMHO you want to see if graalvm is installed already and therefore on an update it should also be considered so that a changed `GRAALVM_VERSION` would apply.

Should we maybe just do an extra case here for GraalVM until this is further clarified instead of scanning entire `extra` folder what will IMHO have undesired implications?